aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1176 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'aerospike-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local nodesPanel = {
```

Comment:
i still have doubts that gabriel was asking for a table with only the most basic information for nodes (no metrics). i know you show'd him your spec (which updated the panels types in the image to "table"), but was it made clear to him that your plan was to only have the list of nodes with no actual metric data?

Reply:
I actually showed Gabriel the exact implementation and got his approval on it.

Reply:
oooh, now i think i see the value. it looks like you have links to the node dashboard? do they automatically select the correct instance selector as well?

Reply:
Not currently, it wasn't clear if he wanted that, so I kept it like this for now while I work on the agent integration.

Reply:
Ok, feel free to resolve....but given the way the tables are, this addition would be worthwhile